app/src/main/java/com/example/gabriel/sociala/editFeedbackActivity.java

```diff
@@ -0,0 +1,63 @@
+package com.example.gabriel.sociala;
+
+import android.support.v7.app.AppCompatActivity;
+import android.os.Bundle;
+import android.view.View;
+import android.widget.Button;
+import android.widget.EditText;
+import android.widget.ImageView;
+import android.widget.TextView;
+
+public class editFeedbackActivity extends AppCompatActivity {
```

Comment:
small nit: can we make it EditFeedbackActivity 
this way we follow java coding styles?

Reply:
Will do!